### PR TITLE
Add SciJava Ops update site

### PR DESCRIPTION
**Pull request by gselzer**
This PR adds the SciJava Ops update site to the main list.

### CODE REVIEW

File: sites.yml [hunk 1]
```diff
@@ -2187,6 +2187,17 @@ sites:
     maintainers:
       - "[Hadrien Mary](https://imagej.net/people/hadim)"
 
+  - name: "Scijava Ops"
```
Comment (ctrueden) [inline sites.yml:2190]: The `j` should always be capitalized when using the term `SciJava`.

File: sites.yml [hunk 2]
```diff
@@ -2187,6 +2187,17 @@ sites:
...
+    id: "Scijava-ops"
```
Comment (ctrueden) [inline sites.yml:2191]: I suggest making the ID be `SciJava-Ops`. It's OK to have additional capital letters. Avoiding spaces for formal IDs is always a good idea, although in practice it doesn't matter here, at least for now.

**Review by ctrueden — CHANGES_REQUESTED** [review]

File: sites.yml (hunk 2 above)
Comment (gselzer) [reply sites.yml]: Fixed in 4fefafa

**Comment by gselzer** [comment]
Thanks for looking over it so quickly @ctrueden! I just pushed a fix to your two suggestions

**Pull request merged**